src/index.js

```diff
@@ -51,6 +52,15 @@ function render(
   }
 }
 
+function TestHook({callback}) {
+  callback()
+  return null
+}
+
+const testHook = callback => {
```

Comment:
Glad to see I'm not alone on this one 😄

Reply:
OK, let's do this...

Here's why I don't like using `function`. It's not immutable. 

If you declare a function like this, then it's able to be changed (intentionally or otherwise).

```js
> function test() { return 'test'; }
> test = 1;
```

If it were a `const`, you couldn't. I don't use `let` either (unless I can help it) for the same reason.

```js
> const test = () => 'test';
> test = 1;
VM619:1 Uncaught TypeError: Assignment to constant variable.
```

The other reason to use `const` with an arrow function assignment is that you can declare it as a *single statement* arrow function, and thus eliminate the `return` statement. Less visual clutter.

I'll change it as it's not my package, but I disagree.

Reply:
Done. see 92b0c38